jest.config.js

```diff
@@ -6,5 +9,13 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/jest-setup.ts'],
   moduleNameMapper: {
     'react-i18next': '<rootDir>/__mocks__/react-i18next.ts',
+    ...pathsToModuleNameMapper(compilerOptions.paths, {
```

Comment:
could just type the alias values out, but this way it will read them from `tsconfig`